src/composer.json

```diff
@@ -8,19 +8,19 @@
         "php": "^8.2",
```

Comment:
Lets update `php` to 8.3 version

Reply:
(then you should update Dockerfile as well)

Reply:
Done.